rllib/utils/policy.py

```diff
@@ -138,9 +139,13 @@ def load_policies_from_checkpoint(
             continue
 
         merged_config = merge_dicts(policy_config, policy_spec.config or {})
+        policy_class = get_tf_eager_cls_if_necessary(
```

Comment:
Also, please add a comment here on why we need to do this. Basically: Why is the policy_spec.policy_class not already pointing to the correct class? We should always just be able to do:
```
my_new_policy = policy_spec.policy_class(some_config)
```

For tracing policies, I understand it, but for EagerV2 policies, this should work, no?

Reply:
EagerV2 also has traced and non-traced versions, right?
Basically, the name of the policy class, for example DQNTF2Policy, plus the configuration, for example eager_tracing=True, would decide here whether we would be using an eager traced policy in inference.